.github/workflows/setup.yml

```
@@ -10,12 +10,10 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        php-versions: [7.3, 7.4]
+        php-versions: [7.4, 8.0, 8.1]
```

```suggestion
        php-versions: ['7.4', '8.0', '8.1']
```

to avoid interpretation as floating numbers